client/src/components/SearchBar/SearchBar.tsx

```diff
@@ -6,7 +6,7 @@ export default function SearchBar({ handleChange }: SearchBarProps) {
     <div className="flex w-full justify-center">
       <label
         htmlFor="search-input"
-        className="flex w-96 gap-4 rounded-lg border border-primary-dark p-1"
+        className="flex w-full gap-4 rounded-lg border border-primary-dark p-1"
```

Comment:
This enables the searchBar to take the dimension of its parent

Reply:
It also modifies the admin section, do we want to pass the width TW class as a prop ? 
![image](https://github.com/user-attachments/assets/709097ac-ade3-4ea3-b401-a45d0a1baf67)